README.md

```diff
@@ -146,3 +146,9 @@ __Q: Is it possible to use Barrier on Mac OS X / OS X versions prior to 10.12?__
 >      - *(see [#109](https://github.com/debauchee/barrier/issues/109) and [#1251](https://github.com/debauchee/barrier/issues/1251) for status or to volunteer your talents)*
 >
 > The complete list of open issues can be found in the ['Issues' tab on GitHub](https://github.com/debauchee/barrier/issues?q=is%3Aissue+is%3Aopen). Help is always appreciated.
+
+**Q: Error in Windows: `ssl certificate doesn't exist: C:\Users\<username>\AppData\Local\Barrier\SSL\Barrier.pem`**
+
+> A: Use WSL2 to install `openssl` and generate SSL key
+>   - `openssl req -x509 -nodes -days 9999 -subj /CN=Barrier -newkey rsa:4096 -keyout Barrier.pem -out Barrier.pem`
```

Comment:
I really appreciate this FAQ item, as it probably saved me hours of looking it up.  Perhaps this note would be better stored in the [wiki / known issues](https://github.com/debauchee/barrier/wiki/Known-Issues)?

Reply:
Just created here: https://github.com/debauchee/barrier-wiki/pull/10